packages/data-flow/src/eventsFetcher.ts

```diff
@@ -19,4 +19,21 @@ export class EventsFetcher implements IEventsFetcher {
             limit,
         );
     }
+
+    /** @inheritdoc */
+    async fetchEventsBySrcAddress(params: {
+        chainId: ChainId;
+        srcAddresses: Address[];
+        from?: {
+            blockNumber?: number;
+            logIndex?: number;
+        };
+        to: {
+            blockNumber: number;
+            logIndex: number;
```

Comment:
should it be like this ?

```suggestion
        from: {
            blockNumber: number;
            logIndex: number;
        };
        to?: {
            blockNumber: number;
            logIndex: number;
```